### PR TITLE
Update the apiserver-aggregation.md

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref: #45409 
Updated the referenced doc as per the issue and the proposed solution

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The additional APIs can either be ready-made solutions such as a
 [metrics server](https://github.com/kubernetes-sigs/metrics-server), or APIs that you develop yourself.
 
 The aggregation layer is different from
-[Custom Resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/),
+[Custom Resources Definitions](/docs/concepts/extend-kubernetes/api-extension/custom-resources/),
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md:19]: Did you mean:
```suggestion
[Custom Resource Definitions](/docs/concepts/extend-kubernetes/api-extension/custom-resources/),
```

Comment (vaibhav2107) [reply content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md:19]: Yes @sftim, thanks for the correction. I will update the PR.

**Comment by mbianchidev** [comment]
@vaibhav2107 any update on that correction?

Other than that, lgtm! (Can only be an informal label-less)

**Comment by vaibhav2107** [comment]
I have updated the PR

**Comment by vaibhav2107** [comment]
@sftim @mbianchidev Could you please have a look now

**Comment by mbianchidev** [comment]
I cannot approve formally but lgtm!

**Pull request merged**